internal/migration/interactive.go

```
@@ -0,0 +1,197 @@
+		if cat.ID == fmt.Sprintf("%d", cfg.GitHub.XenForoNodeID) {
+			categories[i].Name = cat.Name + " ✓ Already migrated"
+		}
+	}
+
+	selectedCategory, err := config.PromptSelection("Select XenForo category to migrate:", categories)
+	if err != nil {
+		return err
+	}
+
+	nodeID := 0
+	fmt.Sscanf(selectedCategory.ID, "%d", &nodeID)
```

golangci-lint (1.64.8)

Error return value of `fmt.Sscanf` is not checked

(errcheck)